ML Baselines/NB-SVM_and_LR-TF-IDF.ipynb

```diff
@@ -0,0 +1,389 @@
+{
```

Comment:
Can you please confirm if these notebooks have been processed by nb-stripout? That'll reduce the notebook size and reduce merge conflicts in future